templates/default/pages/[[...slug]].jsx

```diff
@@ -20,6 +21,9 @@ export async function getStaticProps({ params }) {
 export async function getStaticPaths() {
   const paths = allDocuments.map((page) => {
     const parts = page.url_path.split("/");
+    if (page.isDraft) {
+      return { params: { slug: undefined } };
+    }
     return { params: { slug: parts } };
   });
```

Comment:
Wouldn't it be better to simply filtered out pages for which `page.isDraft` is set to `true`?